src/main/service/manualUploadService.ts

```diff
@@ -52,27 +52,37 @@ export class ManualUploadService {
     return [{text: 'SJP Media Register', value: 'SJP_MEDIA_REGISTER'}];
   }
 
-  public async validateFormFields(formValues: object): Promise<object> {
+  public async validateFormFields(formValues: object, language: string): Promise<object> {
     const fields = {
-      courtError: await this.validateCourt(formValues['input-autocomplete']),
-      contentDateError: this.validateDate(this.buildDate(formValues,'content-date-from')),
-      displayDateError: this.validateDates(this.buildDate(formValues, 'display-date-from'), this.buildDate(formValues, 'display-date-to')),
+      courtError: await this.validateCourt(formValues['input-autocomplete'], language),
+      contentDateError: this.validateDate(this.buildDate(formValues,'content-date-from'), language),
+      displayDateError: this.validateDates(this.buildDate(formValues, 'display-date-from'),
+        this.buildDate(formValues, 'display-date-to'), language),
     };
     if (!fields.courtError && !fields.contentDateError && !fields.displayDateError) {
       return null;
     }
     return fields;
   }
 
-  private async validateCourt(courtName: string): Promise<string> {
+  private async validateCourt(courtName: string, language: string): Promise<string> {
     if (courtName?.length >= 3) {
-      const validCourt = await courtService.getLocationByName(courtName);
+      const validCourt = await courtService.getLocationByName(courtName, language);
       if (validCourt) {
         return null;
       }
-      return 'Please enter and select a valid court';
+
+      if (language == 'cy') {
```

Comment:
It might be a lot of effort to change it, or might not be possible. Just thinking out loud....can those error strings not going into the language resource files and we use the appropriate ones based on req.lng. This should put all translations in one place rather than hardcoding them?